release-notes/release-notes-v10.2.0.md

```diff
@@ -54,9 +66,14 @@ Note: A reparse from block 819250 is automatically launched during the update.
 * `/v2/assets` accepts now the paramater `named=true|false` to return only named or numeric assets. 
 * Publish events on ZMQ Pub/Sub channel (see Documentation).
 * Database connection pooling.
+* If `verbose=true`, enrich results containing `block_index` with `block_time`.
+* Added an `action` filter for the `*/credits` and `*/debits` routes.
+* Add `issuer=None` in XCP and BTC asset information.
+* Excludes zero balances in the results of `/v2/addresses/<address>/balances`.
+* Add BTC sent in `DISPENSE` event.
 
 ## Command-Line Interface
-
+* `-v` for `DEBUG` level, `-vv` for `EVENT` level, `-vvv` for `TRACE` level
```

Comment:
is there a way to do this with `--verbose`?

Reply:
great. I didn't see this in the `--help` text or the docs, but maybe I missed it. would you mind checking?